blog(embertimes118) mike north podcast (#296)
ember-learn/ember-blog: source/2019-10-04-the-ember-times-issue-118.md

---

## [Podcast: The New Ember with Mike North 🎙️](https://devchat.tv/js-jabber/jsj-395-the-new-ember-with-mike-north/)

> A recent [JavaScript Jabber](https://devchat.tv/js-jabber/) podcast had [@mike-north](https://github.com/mike-north) discussing the “New Ember” and how it is a lot more approachable than it used to be. This “New Ember”, or [Ember Octane](https://emberjs.com/editions/octane/), brings Ember from having the steepest learning curve of the popular JavaScript frameworks to hardly having to learn anything at all.

Reviewer: Just for brevity:
```suggestion
A recent [JavaScript Jabber](https://devchat.tv/js-jabber/) podcast had [@mike-north](https://github.com/mike-north) discussing the “New Ember” and how it is a lot more approachable now. This “New Ember”, or [Ember Octane](https://emberjs.com/editions/octane/), brings Ember from having the steepest learning curve of the popular JavaScript frameworks to hardly having to learn anything at all.
```
